docs/tools/tools_intro.md

```diff
@@ -19,7 +19,18 @@ The Mbed Online Compiler is our in-house IDE and should be familiar to anyone wh
 
 For more information, please see the [Online Compiler page](../build-tools/mbed-online-compiler.html).
 
-## Arm Mbed CLI
+## Command line tools
+
+Mbed OS has two versions of its command line tool:
+
+- For 6.y and newer, use the new build tools.
+- For 6.x and older, use Mbed CLI.
```

Comment:
It's not a hard switch. We would support Mbed CLI for 6.y and newer as well. It's only with perhaps 7.x (although not decided yet) that we'd drop Mbed CLI support. We will recommend using the new tools over the old ones in a future Mbed OS release, but not yet.